webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+module.exports = {
```

Comment:
- [ ] Please add some devServer configuration, otherwise the npm start command won't work.
ex:
```
  devServer: {
    publicPath: "/",
    contentBase: "./dist",
    hot: true
  },
```